components/script/dom/webidls/WebGLQuery.webidl

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+//
+// WebGL IDL definitions scraped from the Khronos specification:
+// https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.8
+//
+
+[Exposed=Window]
+interface WebGLQuery : WebGLObject {
```

Comment:
This needs to be hidden behind the webgl2 preference: https://github.com/servo/servo/blob/86d971d1a21772920ca711a3531e2c04bcf60766/components/script/dom/webidls/WebGL2RenderingContext.webidl#L583